mappings/net/minecraft/block/LecternBlock.mapping

```diff
@@ -13,7 +13,7 @@ CLASS net/minecraft/class_3715 net/minecraft/block/LecternBlock
 	FIELD field_17372 SOUTH_SHAPE Lnet/minecraft/class_265;
 	METHOD <init> (Lnet/minecraft/class_2248$class_2251;)V
 		ARG 1 settings
-	METHOD method_17470 openContainer (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;)V
+	METHOD method_17470 openHandledScreen (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;)V
```

Comment:
Same as in Runemoro's review for AbstractFurnaceBlock, `openScreen` would be enough here.